openpathsampling/sample.py

```diff
@@ -746,23 +748,20 @@ class Sample(StorableObject):
     Attributes
     ----------
     replica : int
-        The replica ID to which this Sample applies. The replica ID can also be negative.
+        The replica ID to which this Sample applies. The replica ID can also
+        be negative.
     trajectory : openpathsampling.Trajectory
         The trajectory (path) for this sample
     ensemble : openpathsampling.Ensemble
         The Ensemble this sample is drawn from
-    details : openpathsampling.MoveDetails
-        Object 
-    step : int
-        the Monte Carlo step number associated with this Sample
     """
 
     def __init__(self,
                  replica=None,
                  trajectory=None,
                  ensemble=None,
                  bias=1.0,
-                 details=None,
+                 # details=None,
```

Comment:
I think this (or one of the similar lines) is causes failures now. If you're going to keep the old files working, you'll need to have a way to let it set details even if it isn't used.